src/App.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import './App.css';
 import { Home, About, Contact, Navigation } from './components';
+import {Route} from 'react-router-dom';
 
 const App = () => (
   <div>
     <Navigation />
+    <Route path='/' exact component={Home}/>
```

Comment:
Looks great. Fun fact: did you know that "exact" is the same as exact={true} in this case? You could also explicitly write "exact={false}" or just leave exact out.

Reply:
Haha yes I did know that, leaving off the `={true/false}` makes it implicitly `true`